recipes/dftd3-python/meta.yaml

```diff
@@ -0,0 +1,59 @@
+{% set name = "dftd3-python" %}
+{% set version = "0.4.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/awvwgk/simple-dftd3/archive/refs/tags/v{{ version }}.tar.gz
+  sha256: 79b507a0d892f19222e1403f4c7ebb98ce9eb96b3b8d4821ba5631aa9ac756be
+
+build:
+  number: 0
+  skip: True  # [win or (osx and (python_impl == 'cpython' and py<38))]
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - meson 0.55.*
+    - python
+    - cffi
+    - pkgconfig
+  host:
+    - python
+    - pip
+    - simple-dftd3
+    - libblas
+    - llvm-openmp  # [osx]
+    - libgomp  # [linux and not aarch64]
+  run:
+    - python
+    - cffi
+    - numpy
+    - {{ pin_compatible('simple-dftd3', max_pin='x.x') }}
+
+
+test:
+  imports:
+    - dftd3
+  requires:
+    - pytest
+    - setuptools
```

Comment:
A bit curious why you need `setuptools` here. That isn't a typical test dependency.

Reply:
I'm using `pkg_resources.parse_version` in [one test](https://github.com/awvwgk/simple-dftd3/blob/b33ae37eed3f70c0cef5bad58b31904e6c6c3941/python/dftd3/test_libdftd3.py#L18). I could probably just split the string and save me the trouble in a future version.